docs/en/tutorials/book-store/part-02.md

```diff
@@ -518,14 +518,17 @@ Now you can see the final result on your browser:
 
 ## Create a Books Page
 
-It's time to create something visible and usable! Right click on the `Pages` folder under the {{ if UI == "BlazorServer" }}`Acme.BookStore.Blazor`{{ else }}`Acme.BookStore.Blazor.Client`{{ end }} project and add a new **razor component**, named `Books.razor`:
+It's time to create something visible and usable! Right click on the `Pages` folder under the {{ if UI == "BlazorServer" }}`Acme.BookStore.Blazor`{{ else if UI == "Blazor" }}`Acme.BookStore.Blazor.Client`{{else}} `Acme.BookStore.MauiBlazor` {{ end }} project and add a new **razor component**, named `Books.razor`:
```

Comment:
Is this cover that if the UI selected as `BlazorWebApp`?

Reply:
Covered with https://github.com/abpframework/abp/commit/3f4fa1e22cd12cbaaf0e2351a4c493bab2968fbe